wpeverest/everest-forms
# Enhancement - Export/Import multiple  forms at a time.

### All Submissions:




### Changes proposed in this Pull Request:
Previously, there was no way to export/import multiple  forms at once. This PR provides this feature.

### How to test the changes in this Pull Request:

1. Go to Everest Forms Tools > Export.
2. Select forms  and click on 'Export' button.
3. And import this downloaded json file.
4. Also, verify whether forms are imported/exported properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Export/Import multiple  forms at a time.
